integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -1833,15 +1833,15 @@ def test_std_variance_partial_replace_fallback(data_gen,
         conf=local_conf)
 
 #
-# test min max on single level structure
+# test min/max aggregations for structs
 #
 gens_for_max_min = [byte_gen, short_gen, int_gen, long_gen,
     float_gen, double_gen,
     string_gen, boolean_gen,
     date_gen, timestamp_gen,
     DecimalGen(precision=12, scale=2),
     DecimalGen(precision=36, scale=5),
-    null_gen]
+    null_gen] + array_gens_sample + struct_gens_sample
```

Comment:
You enabled UDTs and CalendarInterval, but I see no tests for those. They are types that we don't really support much. UDT we don't support anywhere right now. CalendarInterval we really only support for literals, but there are a few places we can do more. Also Spark does not support MAPs for sorts at all, which we say we support, but the CPU should fail with.  That last one is a nit, but the others we either need to add tests or stop listing that we support them.

Reply:
I've changed the input type check. Please take a look again.

Reply:
nit: We probably could support binary, but I am fine with this for now. Just file a follow on for binary if we don't do it here.

Reply:
I have no idea if cudf can find min/max on binary type. Filing an issue for it.....

Reply:
Filed https://github.com/NVIDIA/spark-rapids/issues/8643.